lib/phobos_db_checkpoint/handler.rb

```diff
@@ -31,7 +45,15 @@ def around_consume(payload, metadata)
           end
 
           event_action = instrument('db_checkpoint.event_action', event_metadata) do
-            yield
+            begin
```

Comment:
@ticolucci I think this adds complexity. PhobosDBCheckpoint is a plugin for Phobos, if not configured it should keep the original behavior, which is retry forever.